include/uapi/linux/tee.h

```diff
@@ -174,6 +176,16 @@ struct tee_ioctl_buf_data {
 /*
  * These defines shared memory reference parameters (struct
  * tee_ioctl_param_memref)
+ *
+ * A client may need to present a NULL pointer in the argument
+ * passed to a trusted application in the TEE.
+ * This is also a requirement in GlobalPlatform Client API v1.0c
+ * (section 3.2.5 memory references), which can be found at
+ * http://www.globalplatform.org/specificationsdevice.asp
+ *
+ * If a NULL pointer is passed to a TA in the TEE, the ip.c
+ * IOCTL parameters is set to (-1) indicating a NULL memory
```

Comment:
`s/(-1)/TEE_MEMREF_NULL/`

This entire description belongs in the description of the `struct tee_ioctl_param` below.

Reply:
Sure.